routing/payment_session.go

```diff
@@ -243,6 +243,15 @@ func (p *paymentSession) RequestRoute(maxAmt, feeLimit lnwire.MilliSatoshi,
 
 		switch {
 		case err == errNoPathFound:
+			// Don't split if this is a legacy payment without mpp
+			// record.
+			if p.payment.PaymentAddr == nil {
+				p.log.Debugf("not splitting because mpp " +
+					"unsupported")
```

Comment:
`log.Debug` + perhaps rephrase to be more specific, ie "not splitting: no payment addr"  since the dest may support mpp but not provide pmt addr as you mention above?

Reply:
rephrased